README.md

```diff
@@ -53,6 +52,12 @@ https://api.{env}.playtak.com
 - /v1/games-historoy/ptn/{id}
 - /v1/hames-history/db
 
+- /v1/ratings
+- /v1/ratings{player_name}
+
+- /v1/ratings
+- /v1/ratings{player_name}
```

Comment:
Should these have a / in them before the player_name or is that baked into the variable?

Reply:
yeah need to add the / I'll make an update to the readme and double check the endpoints